requirements.txt

```diff
@@ -0,0 +1,8 @@
+kombu
+configman
+taskcluster
+jsonschema
+requests
+arrow
+sh
+redo
```

Comment:
I recommend against using a requirements file. If you're deployed with Puppet you almost certainly want a setup.py instead, so that you can build pip-installable packages. https://github.com/mozilla/buildbot-bridge/blob/master/setup.py is a recent, very simple example.